src/prommis/uky/tests/test_uky_flowsheet.py

```diff
@@ -791,7 +797,7 @@ def test_costing(system_frame):
     )
     assert model.fs.costing.total_fixed_OM_cost.value == pytest.approx(7.2485, rel=1e-4)
     assert model.fs.costing.total_variable_OM_cost[0].value == pytest.approx(
-        10.7422, rel=1e-4
+        1.45469, rel=1e-4
```

Comment:
This is a rather big difference. What changes to cause this?

Reply:
It's the result of changing `m.fs.power` from 14716 hp to 7hp. The former was a value carried over from the commercial scale costing parameters (I forget whether I had already scaled this down or not), and the latter addresses Brandon's previous [comment](https://github.com/prommis/prommis/pull/47#discussion_r1617233106).